Doc/library/math.rst

```diff
@@ -284,6 +286,17 @@ Number-theoretic and representation functions
    :class:`~numbers.Integral` (usually an integer). Delegates to
    :meth:`x.__trunc__() <object.__trunc__>`.
 
+.. function:: ulp(x)
+
+   Return the value of the least significant bit of a float *x*, such that the
+   first float bigger than *x* is ``x + ulp(x)``.  Then, given an expected
+   result *x* and a tolerance of ``n`` ulps, the result ``y`` should be such
+   that ``abs(y - x) <= n * ulp(x)``.
```

Comment:
This gets messy across power of two boundaries; in that case, "a tolerance of n ulps" isn't a well-defined thing without further clarification or specification. It may be better simply to drop this sentence.